### PR TITLE
Add postiz

### PR DESCRIPTION
**Pull request by nevo-david**
## Requirements
* [ ]  This is Software as a Service not self hosted
* [x]  It has a free tier not just a free trial (only in the open-source)
* [x]  Pricing information is clearly visible without signup or phone calls
* [x]  The submission mentions what is free
* [x]  The submission is not already present in the list
* [x]  The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1902,6 +1902,7 @@ Update Time, five active automations, webhooks.
   * [OnlineExifViewer](https://onlineexifviewer.com/) — View EXIF data online instantly for a photo including GPS location and metadata.
   * [PDFMonkey](https://www.pdfmonkey.io/) — Manage PDF templates in a dashboard, call the API with dynamic data, and download your PDF. Offers 300 free documents per month.
   * [Pika Code Screenshots](https://pika.style/templates/code-image) — Create beautiful, customizable screenshots from code snippets and VSCode using the extension.
+  * [Postiz](https://postiz.com) — Self-hosted social media management software. It helps you to schedule posts, track the performance of your content, and manage all your social media accounts in one place.
```
Comment (ripienaar) [inline README.md:1905]: Does this entry tick all the boxes in the PR template?

Comment (nevo-david) [reply README.md:1905]: You are too fast :)
Updated

**Comment by nevo-david** [comment]
Hi @ripienaar, let me know if there are any problems :)

**Comment by ripienaar** [comment]
As stated. The list is for SaaS software.

**Comment by nevo-david** [comment]
Yeah, it's a SaaS and it's selfhosted

**Comment by ripienaar** [comment]
Software as a service is not self hosted software. To qualify the SaaS has to have a free tier.

**Comment by nevo-david** [comment]
What do you mean? Our cloud platform.postiz.com is the same as https://github.com/gitroomhq/postiz-app

**Comment by ripienaar** [comment]
Do you offer a free tier for your SaaS offering that isn’t self hosted?  The SaaS has to have a free tier.

**Comment by nevo-david** [comment]
No, but if you clone the repository, and deploy it on your server everything is free

**Pull request closed without merging**